### ISSUE TITLE
Missing default export

### ISSUE THREAD

**Issue by trajano** [robert8888/mime-file-extension#7]
You had this

```
const mfe = require("mime-file-extension");
```

The TypeScript equivalent is 
```
import mfe from "mime-file-extension");
```

But you don't have a default export so that stopped working.

**Comment by robert8888** [comment]
can you confirm version ? is this v2.0.4 ?

(i had problem with exports in .2 and .3) - but i think that it was solved in 2.0.4.

**Comment by robert8888** [comment]
Fixed in 2.0.6. 

[line](https://github.com/robert8888/mime-file-extension/blob/b2a000aee8d93499768926ca3299bee91d6a9341/src/index.ts#L47)

I had to make export for individual function (to work with commonjs and don't break v1 behavior) and on top of that export default to work with typescript importer.

I briefly tested and seems to work. Can you confirm this ?

**Comment by trajano** [comment]
seems to work. Though I use your exported functions now because I prefer those anyway.
